### ISSUE TITLE
[BUG] Enterprise Login required packages at First Setup missing

### ISSUE THREAD

**Issue by sgohl** [fedora-silverblue/issue-tracker#320]
**Describe the bug**

The First Setup Wizard usually installs the required packages automatically on Fedora Workstation, but at Silverblue you get the following error message (despite having active network connection):

![image](https://user-images.githubusercontent.com/8873017/182681798-d60dc702-2a56-48a1-9a13-e2c788f1c61f.png)

**To Reproduce**

1. install silverblue + reboot
2. click Enterprise Login at First Setup
3. enter Domain credentials
4. click Next

**Expected behavior**

The expected behavior is that Silverblue will add the packages needed

**Screenshots**

**OS version:**
```
44a29b6dff24a71b43e3610ba87f29d5  Fedora-Silverblue-ostree-x86_64-36-1.5.iso
```

**Comment by travier** [comment]
Agree that we should add those to the base image. We kind of need the `domain-client` comps group.

**Comment by sgohl** [comment]
> add those to the base image
not necessarily, to keep the base image small for those who don't need it. Your decision, but just saying.

I guess it's good enough to let it `rpm-ostree install` the packages and notify that a reboot is required to proceed.

The problem here is supposingly that the credentials are already entered and then have to be re-entered a second time after reboot (unless it can remember?). Perhaps it would be possible to let it check for the packages at switching to the Enterprise tab, right before you enter the creds?

**Comment by travier** [comment]
We could use `rpm-ostree install --live` to get them directly in the current boot.

But in this case, this is system functionality that is expected to be part of the base system, not just an application, thus why I think it should be in the base image.

**Comment by tpopela** [comment]
@travier I would rather use the `rpm-ostree install --live` as you've suggested than to add the following (especially samba) to the base image:

```
  libkadm5-1.19.2-11.fc36.x86_64
  lmdb-0.9.29-3.fc36.x86_64
  lv2-1.18.2-2.fc36.x86_64
  oddjob-0.34.7-5.fc36.x86_64
  oddjob-mkhomedir-0.34.7-5.fc36.x86_64
  python3-dns-2.2.0-1.fc36.noarch
  python3-ldb-2.5.2-1.fc36.x86_64
  python3-samba-2:4.16.4-0.fc36.x86_64
  python3-samba-dc-2:4.16.4-0.fc36.x86_64
  python3-talloc-2.3.4-1.fc36.x86_64
  python3-tdb-1.4.7-1.fc36.x86_64
  python3-tevent-0.12.1-1.fc36.x86_64
  samba-2:4.16.4-0.fc36.x86_64
  samba-common-tools-2:4.16.4-0.fc36.x86_64
  samba-dc-libs-2:4.16.4-0.fc36.x86_64
  samba-libs-2:4.16.4-0.fc36.x86_64
  tdb-tools-1.4.7-1.fc36.x86_64
```

**Comment by travier** [comment]
Related but independent decision: We're considering adding samba to KDE/Kinoite by default to let users easily share directories on the network (https://pagure.io/fedora-kde/SIG/issue/109). Right now it's blocked by the following bug: https://bugzilla.redhat.com/show_bug.cgi?id=2096405.

**Comment by travier** [comment]
It's been a while here. Could someone take a look at which packages are missing again and how much they would add to the image?

**Comment by travier** [comment]
From https://discussion.fedoraproject.org/t/bug-in-f40-packages-missing-for-ad-integration/112410/2:
```
rpm-ostree install adcli sssd-ad oddjob-mkhomedir samba-common-tools
```

**Comment by travier** [comment]
See:
- https://gitlab.com/fedora/ostree/sig/-/issues/24
- https://github.com/fedora-silverblue/issue-tracker/issues/550
- https://pagure.io/workstation-ostree-config/pull-request/504

**Comment by woprandi** [comment]
I see this issue and https://github.com/fedora-silverblue/issue-tracker/issues/550 are still open. Is something still blocking ?

**Comment by travier** [comment]
It's unlikely that we'll add them by default to the image. You should be able to add them to the container image and use it instead. Or we could make a sysext for them: https://github.com/travier/fedora-sysexts

**Comment by woprandi** [comment]
I was not able to override, I don't understand why : https://github.com/osbuild/bootc-image-builder/issues/983
